frontend/src/router/index.ts

```diff
@@ -57,6 +64,7 @@ const routes = [
   },
   {
     path: "/:catchAll(.*)",
+    meta: withMeta({requiresAuth: false}),
```

Comment:
As mentioned below, this seems wrong to have the withMeta function default apply auth but it is unset here.